.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Brief description of changes.
+
+## Type of Change
+
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Documentation update
+- [ ] ADR (Architecture Decision Record)
+
+## ADR Checklist (if applicable)
+
+If this PR includes a new ADR, ensure:
+
+- [ ] ADR is added to `mkdocs.yml` nav section
+- [ ] ADR is added to "Recent Decisions" table in `docs/index.md`
+- [ ] ADR follows the standard template
```

Comment:
This duplicates the ADR PR template; I suggest removing the redundancy.

Related question: How can I use the ADR PR template to create a PR? I don't think GitHub supports multiple PR templates, does it?

Reply:
Yes that  is correct, you have to manually construct a URL to use multiple PR templates. I will delete the adr.md